src/mmrelay/migrate.py

```
@@ -0,0 +1,1246 @@
+                    config_data = yaml.safe_load(f)
+                    if isinstance(config_data, dict):
+                        plugins_section = config_data.get("community-plugins", {})
+                        if isinstance(plugins_section, dict):
+                            gpx_section = plugins_section.get("gpxtracker", {})
+                            if isinstance(gpx_section, dict) and gpx_section.get(
+                                "gpx_directory"
+                            ):
+                                gpx_configured = True
+                                break
+            except Exception:
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)